core/src/main/resources/db/cgds.sql

```diff
@@ -177,16 +178,34 @@ CREATE TABLE `sample_list_list` (
 );
 
 -- --------------------------------------------------------
+CREATE TABLE `genetic_entity` (
+  `INTERNAL_ID` int(11) NOT NULL AUTO_INCREMENT,
```

Comment:
Can we call this `ID`? I know we have lots of tables using the name `INTERNAL_ID`, but it's because they have other kinds of IDs in the same table, like STABLE_ID. But this table doesn't have that problem.